sql-parser/statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowTablesStatement.java

```diff
@@ -36,6 +36,8 @@ public final class MySQLShowTablesStatement extends AbstractSQLStatement impleme
     
     private ShowFilterSegment filter;
     
+    private Boolean hasFull;
```

Comment:
Why not use boolean? And what about constainsFull?

Reply:
done.